Missionframework/modules/02_core/fnc/fn_core_handleSector.sqf

```diff
@@ -31,8 +31,14 @@ private _handler = [
         // If there are no enemy units in two times the capture range and friendly units are in capture range
         // capture the sector
         if (
-            !([_sectorPos, 2 * KPLIB_param_sectorCapRange, [KPLIB_preset_sideE]] call KPLIB_fnc_core_areUnitsNear)
-            && {[_sectorPos, KPLIB_param_sectorCapRange] call KPLIB_fnc_core_areUnitsNear}
+            (
+                ({side _x isEqualTo KPLIB_preset_sideF} count (_sectorPos nearEntities ["AllVehicles", KPLIB_param_sectorCapRange])) >
+                ({side _x isEqualTo KPLIB_preset_sideE} count (_sectorPos nearEntities ["AllVehicles", KPLIB_param_sectorCapRange * 1.5]))
+                * KPLIB_param_sectorCapRatio
+            ) && (
+                ({side _x isEqualTo KPLIB_preset_sideE} count (_sectorPos nearEntities ["Tank", KPLIB_param_sectorCapRange * 1.5]))
+                isEqualTo 0
+            )
```

Comment:
Wrap the second condition into braces instead of parenthesis. There is no need to do 3 x nearEntities each time. I have a feeling that this whole code block might be optimized more, maybe consider adding `// TODO optimize` above the if.